ui/components/ui/tab-empty-state/tab-empty-state.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { TabEmptyState } from './tab-empty-state';
+import { useTheme } from '../../../hooks/useTheme';
+
+// Theme-aware image component for perps trading
+const TradingIcon = () => {
+  const theme = useTheme();
+  const imageSrc =
+    theme === 'light'
+      ? './images/empty-state-perps-light.png'
+      : './images/empty-state-perps-dark.png';
+
+  return (
+    <img
+      src={imageSrc}
+      alt="Trading perps"
+      style={{ width: '64px', height: '64px' }}
+    />
+  );
+};
```

Comment:
The images don’t show correctly in Storybook because the `useTheme()` hook always prioritizes the Redux store value over the `data-theme` attribute. Since the store has a default theme set, it ignores Storybook’s theme toggle. This will be resolved when the images are used with the feature and app state itself.